src/sentry/static/sentry/app/components/asyncComponent.jsx

```diff
@@ -20,6 +20,24 @@ class AsyncComponent extends React.Component {
     router: PropTypes.object,
   };
 
+  static errorHandler(component, fn) {
+    return (...args) => {
+      try {
+        return fn(...args);
+      } catch (err) {
+        /*eslint no-console:0*/
```

Comment:
Can you change this to `// eslint-disable-next-line no-console` while you're in here? I think this disables the rule for the entire file?